tests/unit/backend/data/test_GroceryService.py

```diff
@@ -65,10 +67,31 @@ def setUpClass(cls):
         # create some sample data
         cls.sampleWS = "_grocery_to_fetch"
 
-        LoadEmptyInstrument(
-            Filename=cls.instrumentFilePath,
+        CreateSampleWorkspace(
+            OutputWorkspace=cls.sampleWS,
+        )
+        CreateSampleWorkspace(
```

Comment:
Can you time how much this slows down unit tests (if at all)?  My concern is this workspace is used in an awful lot of tests.